WindowsServerDocs/administration/OpenSSH/OpenSSH_KeyManagement.md

```diff
@@ -141,22 +141,22 @@ After completing these steps, whenever a private key is needed for authenticatio
 
 ## Deploying the public key
 
-To use the user key that was created above, the public key needs to be placed on the server into a text file called *authorized_keys* under users\username\.ssh\. 
+To use the user key that was created above, the public key needs to be placed on the server into a text file called *authorized_keys* under `C:\Users\username\.ssh\`. 
 The OpenSSH tools include scp, which is a secure file-transfer utility, to help with this.
 
-To move the contents of your public key (~\.ssh\id_ed25519.pub) into a text file called authorized_keys in ~\.ssh\ on your server/host.
+To move the contents of your public key (`~\.ssh\id_rsa.pub`) into a text file called authorized_keys in `~\.ssh\` on your server/host.
```

Comment:
Maybe I don't understand how the procedure works, but it feels like this sentence is unfinished by not saying "what needs to be done" after the current content.

Reply:
I've only changed the name of the file and added inline backticks to format paths as a code and avoid publishing procedure to mess up with "\." and remove "\".

Reply:
Very well, fair enough. I just thought maybe it could be useful to find out if the sentence was supposed to describe what kind of action is needed "to move the contents of your public key into a text file called authorized_keys on your server/host", because that logic seems to be missing. (I could very well be mistaken, I don't know this topic.)

Reply:
You are correct that that sentence doesn't sound like it's finished. Because it actually isn't. :(
It should be attached to the previous sentence. However, this whole section need to be rewritten because steps that are described are incorrect and won't allow a reader to authenticate using key files.

Reply:
Thank you for the detailed reply. Fair enough. Maybe we could ask @maertendMSFT for a suggestion on how to deal with this particular question.